libs/qscintilla/scintilla/lexlib/LexAccessor.h

```diff
@@ -163,7 +163,7 @@ class LexAccessor {
 	void ColourTo(Sci_PositionU pos, int chAttr) {
 		// Only perform styling if non empty range
 		if (pos != startSeg - 1) {
-			assert(pos >= startSeg);
+			//assert(pos >= startSeg);
```

Comment:
Oh, was this commented out upstream, or a manual change? :smile:

Reply:
That's an upstream change. 

The only change that I have made compared to the original tarball is to remove most lexers, matching the current selection.

To be precise:
* delete several `scintilla/lexers/Lex*.cpp`, `src/Qsci/qscilexer*.h`, `src/qscilexer*.cpp`
* remove entries from `scintilla/src/Catalogue.cpp`
* Write custom CMakeLists.txt